src/measure_theory/probability_mass_function.lean

```diff
@@ -8,27 +8,31 @@ Probability mass function -- discrete probability measures
 import topology.instances.ennreal
 noncomputable theory
 variables {α : Type*} {β : Type*} {γ : Type*}
-open_locale classical big_operators
+open_locale classical big_operators nnreal
```

Comment:
Could you please add a very short module docstring explaining that this file does not use `measure_theory.measure` in any way and this fact should be fixed some day.